src/Core/Prefab/GlobeView.js

```diff
@@ -384,13 +385,24 @@ GlobeView.prototype.screenCoordsToNodeId = function screenCoordsToNodeId(mouse)
     return Math.round(unpack);
 };
 
+GlobeView.prototype.getDepthBuffer = function getDepthBuffer(x, y, width, height) {
```

Comment:
I'd prefer `readDepthBuffer` since `getXXX` is reserved for trivial getter functions.